### PR TITLE
Merging code changes and text file changes with Development

### LINKED ISSUE #197: Create an Award ceremony in the training
* **Is your feature request related to a problem? Please describe.**

This training resources would benefit from awards

* **Describe the solution you'd like**

please create some awards

* **Describe alternatives you've considered**

doing it via email

* **Additional context**

### PR DESCRIPTION
**Pull request by EwanDevineKennedy**
## Proposed changes

changing teh min calculation to Mean calcualtion 

## Types of changes

What types of changes does your code introduce?

changing teh min calculation to Mean calcualtion 

- [ ] Bugfix (change which fixes an issue)
- [ X] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [X ] I have added necessary documentation (if appropriate)
- [ X] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

Tag your reviewer here to ensure they get a notification!
@AIMS-Edisa-Livingstone

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -3,3 +3,8 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 This is some other text.
+
+This is an change ot teh test
+mary had a little lamb
+which was very tasty
+with horseradish sauce and potatoes
```
Comment (isabellaimage) [inline text_file.txt]: Personally I'm not keen on horseradish

**Review by isabellaimage — CHANGES_REQUESTED** [review]
Change of condiments needed

**Review by AIMS-Edisa-Livingstone — APPROVED** [review]
Fine, nothing to see here of concern

**Review by AIMS-Edisa-Livingstone — APPROVED** [review]
Always a good idea to bake some spuds

File: text_file.txt (hunk 1 above)
Comment (EwanDevineKennedy) [reply text_file.txt]: Ive amendedto just potatoes

https://github.com/department-for-transport/git-and-github-training/blob/5235207687590d808cd50a0eea64491f30d342ba/text_file.txt#L6-L10

Comment (isabellaimage) [reply text_file.txt]: oh dear I feel bad now, I hope you have mustard or cranberry sauce.  Anyway will now approve

**Review by isabellaimage — APPROVED** [review]

**Pull request closed without merging**